src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawing {
```

Comment:
All your figures should have one more common behavior: we can obtain the area of any figure. Implement this functionality